std_msgs/String message
std_msgs/Bool ok